cob_bringup/robots/cob4-3.xml

```diff
@@ -23,25 +23,6 @@
 		</include>
 
 		<!-- startup hardware -->
-		<include file="$(find cob_bringup)/drivers/sick_s300.launch" >
-			<arg name="robot" value="$(arg robot)" />
-			<arg name="name" value="base_laser_front" />
-		</include>
-		<include file="$(find cob_bringup)/drivers/sick_s300.launch" >
-			<arg name="robot" value="$(arg robot)" />
-			<arg name="name" value="base_laser_right" />
-		</include>
-		<include file="$(find cob_bringup)/drivers/sick_s300.launch" >
-			<arg name="robot" value="$(arg robot)" />
-			<arg name="name" value="base_laser_left" />
-		</include>
-		<include file="$(find cob_bringup)/drivers/sick_flexisoft.launch" >
-			<arg name="robot" value="$(arg robot)" />
-		</include>
-		<include file="$(find cob_bringup)/drivers/scan_unifier.launch" >
-			<arg name="robot" value="$(arg robot)" />
-		</include>
-
```

Comment:
They have still scanners(only 2) and flexisoft but the devices are not connected to the base computer, they have their own system to manage the sick components